apps/src/clientApi.js

```diff
@@ -1,38 +1,71 @@
-/* global dashboard */
 // TODO(dave): Merge with the client API in /shared.
```

Comment:
We merged the NetSim API at one point, but then it diverged again.  At this point the NetSim API should not be merged with the client API - it does some special things that the client API doesn't need to do.